configure.ac

```diff
@@ -673,6 +677,26 @@ if test "x$included_pcre2" = "xyes"; then
   LIBS="$LIBS $PCRE2_LIBS"
 fi
 
+
+
+# ========
+# MuParser math expression library configuration.
+EXTRA_MUPARSER=
```

Comment:
Understood but this is just a baby step. Refinements like using the system provided library can happen later once we're confident this is our preferred solution.